source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,37 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
```

Comment:
Should e be in quotes? Should the brackets be the same? Right now, it is mixed with squares/parentheses.

Reply:
This `e` thing should be removed, but it's better done in a separate patch.

Reply:
Thank you! I did not know that about the brackets and will keep that in mind for future reviews.

Reply:
> This e thing should be removed, but it's better done in a separate patch.

Also added to #565.